packages/compose/README.md

```diff
@@ -116,9 +116,7 @@ _Usage_
 ```ts
 type Props = { foo: string };
 const Component = ( props: Props ) => <div>{ props.foo }</div>;
-const ConditionalComponent = ifCondition(
-	( props: Props ) => props.foo.length !== 0
-)( Component );
+const ConditionalComponent = ifCondition( ( props: Props ) => props.foo.length !== 0 )( Component );
```

Comment:
@tyxla Did you just manually format the `.md` file to create this change? Maybe we should automatically format all `.md` files.

Reply:
No, I formatted it automatically running `npm run other:check-local-changes` locally. No other changes were created.

Reply:
Re-added the commit since it seemed to be lost during the last rebase.